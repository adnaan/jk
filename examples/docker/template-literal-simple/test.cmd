jk generate --stdout -f my-service.yaml dockerfile.js
