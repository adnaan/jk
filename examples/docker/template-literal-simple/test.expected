FROM alpine:3.8

EXPOSE 80

COPY my-service /
ENTRYPOINT /my-service
